### PR TITLE
Add checkboxes to "be a reviewer" modal

### LINKED ISSUE #576: Adicionar checks com os disclaimers de conflito de interesse na solicitação do curador
## Feature Description

Adicionar os seguintes disclaimers na solicitação para ser um curador os itens obrigatórios, data e hora do cadastro.

seja-curador

- [ ]  Declaro estar disponível para avaliar as informações cadastradas pelos usuários no cadastro de tecnologias, checar sua coerência e consistência, bem como analisar o nível das tecnologias cadastradas na Plataforma Sabiá, desde que estejam dentro da minha área de especialidade. (OBRIGATÓRIA)
- [ ]  Declaro que NÃO POSSUO conflito de interesses de ordens financeira, comercial, política, acadêmica e pessoal com as tecnologias pela qual irei avaliar. (OBRIGATÓRIA)
- [ ]  Declaro que irei atender aos princípios éticos no uso da Plataforma Sabiá e desempenharei as minhas funções com probidade, boa-fé e de acordo com a legislação vigente, sendo inteiramente responsável pelas condutas lesivas a direitos de terceiros, ao próprio serviço disponibilizado pela plataforma e à Administração Pública, com responsabilização civil contratual e/ou extracontratual e adoção das medidas penais aplicáveis ao caso, conforme os Termos e Condições de Uso. (OBRIGATÓRIA)

- A fonte desses disclaimers pode ser menor.
- Segue os mesmos moldes da issue #465 

Depende da Issue #575 para integrar com a API
---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by calaca**
## Summary

Resolves #576 

Peek 2020-11-01 21-00

## QA Steps

1. Login as a user that can request to be a reviewer
2. Go to the user's profile
3. Click "be a reviewer" button
4. Verify that you can submit the form only if there is at least one category + subcategory selected and all checkboxes are checked

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

**Comment by luizeboli** [comment]
Hey @calaca, theres some snapshots outdated

**Comment by calaca** [comment]
> Hey @calaca, theres some snapshots outdated

@luizeboli thanks for the heads up! I haven't noticed that.

File: packages/web/components/Modal/BeAReviewerModal/index.js [hunk 1]
```diff
@@ -54,6 +61,20 @@ const BeAReviewerModal = ({ closeModal }) => {
...
+		if (!type || !types.some((item) => item === type)) {
+			return null;
+		}
+
+		setAcceptedTerms({
+			...acceptedTerms,
+			[type]: !acceptedTerms[type],
+		});
+	};
+
```
Comment (luizeboli) [inline packages/web/components/Modal/BeAReviewerModal/index.js:77]:
```suggestion
	const handleAcceptedTerms = (type) => {
		const types = Object.keys(acceptedTerms);

		if (!type || !types.some((item) => item === type)) {
			return null;
		}

		return setAcceptedTerms({
			...acceptedTerms,
			[type]: !acceptedTerms[type],
		});
	};

```

There's no need to disable this rule, just return

Comment (luizeboli) [inline packages/web/components/Modal/BeAReviewerModal/index.js:77]:
```suggestion
	const handleAcceptedTerms = (type) => {
		const types = Object.keys(acceptedTerms);

		if (!type || !types.some((item) => item === type)) {
			return null;
		}

		return setAcceptedTerms({
			...acceptedTerms,
			[type]: !acceptedTerms[type],
		});
	};

```

There's no need to disable this rule, just return

File: packages/web/components/Modal/BeAReviewerModal/index.js [hunk 2]
```diff
@@ -232,7 +253,66 @@ const BeAReviewerModal = ({ closeModal }) => {
...
+			<CheckBoxField
+				data-testid="conflicts"
+				name="conflicts"
+				value={acceptedTerms.conflicts}
+				onChange={() => handleAcceptedTerms('conflicts')}
+				label={
+					<span>
+						Declaro que NÃO POSSUO conflito de interesses de ordens financeira,
+						comercial, política, acadêmica e pessoal com as tecnologias pela qual irei
+						avaliar.
```
Comment (luizeboli) [inline packages/web/components/Modal/BeAReviewerModal/index.js:280]:
```suggestion
						Declaro que NÃO POSSUO conflito de interesses de ordens financeira,
						comercial, política, acadêmica e pessoal com as tecnologias pelas quais irei
						avaliar.
```

File: packages/web/components/Modal/BeAReviewerModal/index.js [hunk 3]
```diff
@@ -232,7 +253,66 @@ const BeAReviewerModal = ({ closeModal }) => {
...
+				value={acceptedTerms.ethics}
+				onChange={() => handleAcceptedTerms('ethics')}
+				label={
+					<span>
+						Declaro que irei atender aos princípios éticos no uso da Plataforma Sabiá e
+						desempenharei as minhas funções com probidade, boa-fé e de acordo com a
+						legislação vigente, sendo inteiramente responsável pelas condutas lesivas a
+						direitos de terceiros, ao próprio serviço disponibilizado pela plataforma e
+						à Administração Pública, com responsabilização civil contratual e/ou
+						extracontratual e adoção das medidas penais aplicáveis ao caso, conforme os{' '}
```
Comment (luizeboli) [inline packages/web/components/Modal/BeAReviewerModal/index.js:297]:
```suggestion
						Declaro que irei atender aos princípios éticos no uso da Plataforma Sabiá e
						desempenharei as minhas funções com probidade, boa-fé e de acordo com a
						legislação vigente, sendo inteiramente responsável pelas condutas lesivas a
						direitos de terceiros, ao próprio serviço disponibilizado pela plataforma e
						à Administração Pública, com responsabilização civil contratual e/ou
						extracontratual, e adoção das medidas penais aplicáveis ao caso, conforme os{' '}
```

**Review by luizeboli — CHANGES_REQUESTED** [review]
Great @calaca 😃 

Did some comments, check it out

File: packages/web/components/Modal/BeAReviewerModal/index.js (hunk 3 above)
Comment (luizeboli) [inline packages/web/components/Modal/BeAReviewerModal/index.js:297]:
```suggestion
						Declaro que irei atender aos princípios éticos no uso da Plataforma Sabiá e
						desempenharei as minhas funções com probidade, boa-fé e de acordo com a
						legislação vigente, sendo inteiramente responsável pelas condutas lesivas a
						direitos de terceiros, ao próprio serviço disponibilizado pela plataforma e
						à Administração Pública, com responsabilização civil contratual e/ou
						extracontratual, e adoção das medidas penais aplicáveis ao caso, conforme os{' '}
```

Improving grammar

File: packages/web/components/Modal/BeAReviewerModal/index.js [hunk 4]
```diff
@@ -54,6 +61,19 @@ const BeAReviewerModal = ({ closeModal }) => {
...
+	const handleAcceptedTerms = (type) => {
+		const types = Object.keys(acceptedTerms);
+
+		if (!type || !types.some((item) => item === type)) {
+			return null;
+		}
+
+		return setAcceptedTerms({
+			...acceptedTerms,
+			[type]: !acceptedTerms[type],
```
Comment (lcnogueira) [inline packages/web/components/Modal/BeAReviewerModal/index.js:73]: 👌

File: packages/web/components/Modal/BeAReviewerModal/index.js [hunk 5]
```diff
@@ -232,7 +252,66 @@ const BeAReviewerModal = ({ closeModal }) => {
...
+					</span>
+				}
+				required
+			/>
+
+			<S.Button
+				disabled={
+					!selectedValues.length ||
+					!Object.values(acceptedTerms).every((value) => value === true)
+				}
```
Comment (lcnogueira) [inline packages/web/components/Modal/BeAReviewerModal/index.js:312]: 👌

**Review by lcnogueira — APPROVED** [review]
Hey, Lorena! You rock! 👩‍🎤

**Review by luizeboli — APPROVED** [review]

**Pull request merged**